distribution/lib/Standard/Base/0.0.0-dev/src/Meta/Enso_Project.enso

```diff
@@ -91,6 +91,9 @@ type Project_Description
     namespace : Text
     namespace self = self.ns
 
+    ## PRIVATE
+    enso_project_builtin module = @Builtin_Method "Project_Description.enso_project_builtin"
```

Comment:
Please don't put `Builtin_Method`s into non-`private` modules.

Reply:
@radeusgd has a good point. The corresponding builtin method is already defined as [EnsoProjectNode](https://github.com/enso-org/enso/blob/f0e9616d39d4fc24c8074bd4681fed182d0ede00/engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/EnsoProjectNode.java). Thus, this code does not introduce any changes. It seems that [BuiltinFunction.isAutoRegister](https://github.com/enso-org/enso/blob/7b0759f8b3b7ab4276a8668642dbd4b9586fafdd/engine/runtime/src/main/java/org/enso/interpreter/runtime/builtin/BuiltinFunction.java#L16) makes it possible to defined a builtin method in Java without any *shadow* definition in Enso code. I think having these kind of *shadow* definitions is beneficial, as we can immediately see all the definitions of builtin methods on a type.